frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/capacity-and-nodes-step/capacity-and-nodes-step.tsx

```diff
@@ -126,19 +89,8 @@ const SelectCapacityAndNodes: React.FC<SelectCapacityAndNodesProps> = ({
             )}
           />
         </GridItem>
-        <GridItem span={10} className="odf-capacity-and-nodes__select-nodes">
-          <ListPage
-            kind={NodeModel.kind}
-            showTitle={false}
-            ListComponent={InternalNodeTable}
-            nameFilterPlaceholder={t('ceph-storage-plugin~Search by node name...')}
-            labelFilterPlaceholder={t('ceph-storage-plugin~Search by node label...')}
-            customData={{
-              onRowSelected,
-              nodes: new Set(nodes.map(({ uid }) => uid)),
-            }}
-          />
-          {!!nodes.length && <NodesDetails nodes={nodes} />}
+        <GridItem span={10}>
+          <SelectNodesTable nodes={nodes} onRowSelected={onRowSelected} />
```

Comment:
One select table used at both:
1. capcaity and nodes step
2. create local volume set step
Drop usage of different tables